VagrantProvisionCentOS7.sh

```diff
@@ -35,45 +30,27 @@ sudo yum -y install epel-release >> CentOS_upgrade.txt 2>&1
 
 # add the Postgres repo
 echo "### Add Postgres repo ###" > CentOS_upgrade.txt
-sudo rpm -Uvh http://yum.postgresql.org/9.5/redhat/rhel-7-x86_64/pgdg-centos95-9.5-3.noarch.rpm >> CentOS_upgrade.txt 2>&1
+sudo rpm -Uvh https://download.postgresql.org/pub/repos/yum/9.5/redhat/rhel-7-x86_64/pgdg-centos95-9.5-3.noarch.rpm  >> CentOS_upgrade.txt 2>&1
 
 echo "Updating OS..."
 echo "### Update ###" >> CentOS_upgrade.txt
 sudo yum -q -y update >> CentOS_upgrade.txt 2>&1
 echo "### Upgrade ###" >> CentOS_upgrade.txt
 sudo yum -q -y upgrade >> CentOS_upgrade.txt 2>&1
 
-
-echo "### Setup NTP..."
-sudo yum -q -y install ntp
-sudo chkconfig ntpd on
-#TODO: Better way to do this?
-sudo systemctl stop ntpd
-sudo ntpd -gq
-sudo systemctl start ntpd
-
 # Make sure that we are in ~ before trying to wget & install stuff
 cd ~
 
-echo "### Installing an ancient version of NodeJS"
-if [ -f "./${NODEJS_RPM}" ]; then
-    sudo yum -y install ./$NODEJS_RPM
-else
-    wget --quiet $NODEJS_URL/$NODEJS_RPM
-    sudo yum -y install ./$NODEJS_RPM
-fi
+echo "### Installing the repo for an ancient version of NodeJS"
+curl --silent --location https://rpm.nodesource.com/setup | sudo bash -
 
-echo "### Installing an ancient version of NodeJS development files"
-if [ -f "./${NODEJS_DEVEL_RPM}" ]; then
-    sudo yum -y install ./$NODEJS_DEVEL_RPM
-else
-    wget --quiet $NODEJS_URL/$NODEJS_DEVEL_RPM
-    sudo yum -y install ./$NODEJS_DEVEL_RPM
-fi
+echo "### Installing an ancient version of NodeJS"
+sudo yum install -y \
+  nodejs-0.10.46 \
+  nodejs-devel-0.10.46 \
+  yum-plugin-versionlock
```

Comment:
The code you are reverting is designed use a local copy of a package in preference to downloading it again.  This is also important on systems that do not and can not connect to the internet. I'd like to discuss this more with you prior to merging the changes.

Reply:
The code I'm reverting _didn't work_.   As I stated in #1845, there's _not_ a v0.10.48 release of NodeJS from nodesource.  Please point me to the correct distribution that was intended.

Reply:
Alright found the source of confusion.  Legacy 0.10 script points to this repo, which does _not_ have 0.10.48:

https://rpm.nodesource.com/pub/el/7/x86_64/

Whereas the updated repository does:

https://rpm.nodesource.com/pub_0.10/el/7/x86_64/

I'd prefer a way to keep the gpg-verification of the package, but will go 

I'd gotten myself confused over the URL sources, however, the current script in develop still doesn't work due to `wget` being invoked before being installed (`curl` is present on base CentOS 7, `wget` on base Ubuntu).  Thus, I reverted to stable state that I knew worked.